blackbox-exporter/version.txt

```diff
@@ -1 +1 @@
-0.25.0-8
+0.25.0-9
```

Comment:
is it because of apk add --no-cache ca-certificates  i dont see apk update --no-cache

Reply:
The build is going to fail when I revert this change.

Reply:
It failed. https://app.circleci.com/pipelines/github/astronomer/ap-vendor/4427/workflows/7192c174-6cfe-4964-9a87-3eb4a23f8c3b